applications/management/commands/expire_applications.py

```diff
@@ -6,34 +6,36 @@
 
 
 from applications import models, emails
+from applications.views import VIEW_APPLICATION_TYPE
 
 
 class Command(BaseCommand):
     help = 'Checks invites that have expired and sends reminders 24 before'
 
     def handle(self, *args, **options):
         fourdaysago = timezone.now() - timedelta(days=4)
+        msgs = []
         self.stdout.write('Checking reminders...')
-        reminders = models.Application.objects.filter(
-            status_update_date__lte=fourdaysago, status=models.APP_INVITED)
-        self.stdout.write('Checking reminders...%s found' % reminders.count())
+        for Application in VIEW_APPLICATION_TYPE.values():
```

Comment:
This includes Sponsors! No go for me! You should not send an email to sponsors saying they need to register!

Reply:
This not includes Sponsors, Sponsors are accepted, so they cannot be invited or last_reminder. (But i'll make it more eficient making that django doesn't access the db of sponsors)